### PR TITLE
fix: wallet and third party services screens

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Rework the wallet screen](https://trello.com/c/BpE1YTTS/101-rework-the-wallet-screen)

File: new-lamassu-admin/src/components/inputs/base/SecretInput.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { TextInput } from '../base'
+
+const SecretInput = memo(({ value, onFocus, onBlur, ...props }) => {
+  const [focused, setFocused] = useState(false)
+
+  const placeholder = '⚬ ⚬ ⚬ This field is set ⚬ ⚬ ⚬'
+  const previouslyFilled = !!value
+  const tempValue = previouslyFilled ? '' : value
+
+  const iOnFocus = event => {
```
Comment (liordino) [inline new-lamassu-admin/src/components/inputs/base/SecretInput.js]: On other pages there where other variables with a 'i' as prefix but they were renamed. I didn't understood why it wasn't the same on this page. What does thsi 'i' mean? The same occurs on other pages below.

File: new-lamassu-admin/src/pages/Notifications/sections/CryptoBalanceOverrides.js [hunk 2]
```diff
@@ -56,12 +56,10 @@ const CryptoBalanceOverrides = ({ section }) => {
     [LOW_BALANCE_KEY]: Yup.number()
       .integer()
       .min(0)
-      .max(99999999)
```
Comment (liordino) [inline new-lamassu-admin/src/pages/Notifications/sections/CryptoBalanceOverrides.js]: This is the same that we talked about on other review and will be removed, right?

Comment (RafaelTaranto) [reply new-lamassu-admin/src/pages/Notifications/sections/CryptoBalanceOverrides.js]: Yup, should be removed already: https://github.com/lamassu/lamassu-server/pull/395/commits/a5182d76266de51cf6c3983d4dd90022b2a6b29b

File: new-lamassu-admin/src/components/inputs/base/SecretInput.js (hunk 1 above)
Comment (RafaelTaranto) [reply new-lamassu-admin/src/components/inputs/base/SecretInput.js]: Good point, it's a bad practice for sure. Originally I was doing innerOnFocus and such, eventually got replaced by i for inner =p

Comment (RafaelTaranto) [reply new-lamassu-admin/src/components/inputs/base/SecretInput.js]: I think I managed to change them all, let me know if you find some lost in the codebase.

File: new-lamassu-admin/src/routing/routes.js [hunk 3]
```diff
@@ -1,133 +1,140 @@
...
-    <Route path="/maintenance/funding" component={Funding} />
-    <Route path="/maintenance/server-logs" component={ServerLogs} />
-    <Route path="/transactions" component={Transactions} />
     <Route path="/register" component={AuthRegister} />
-    <Route path="/maintenance/machine-status" component={MachineStatus} />
-    <Route path="/compliance/customers" component={Customers} />
+    {flattened.map(({ route, component: Page, key }) => (
+      <Route path={route} key={key}>
+        <Page name={key} />
+      </Route>
```
Comment (mautematico) [inline new-lamassu-admin/src/routing/routes.js:137]: wow, such flat.

File: new-lamassu-admin/src/components/tables/DataTable.js [hunk 4]
```diff
@@ -86,18 +70,22 @@ const ExpRow = ({
...
   data = [],
   Details,
   className,
+  expandable,
   ...props
 }) => {
   const [expanded, setExpanded] = useState(null)
 
+  const coreWidth = R.compose(R.sum, R.map(R.prop('width')))(elements)
+  const expWidth = 1200 - coreWidth
```
Comment (mautematico) [inline new-lamassu-admin/src/components/tables/DataTable.js:84]: noice, so now we don't have to manually reserve extra space for the expand column

**Review by mautematico — COMMENTED** [review]
looks good for me

File: new-lamassu-admin/src/components/editableTable/Row.js [hunk 5]
```diff
@@ -1,83 +1,178 @@
...
+    onEdit,
+    enableEdit,
+    enableDelete,
+    disableRowEdit,
+    onDelete,
+    deleteWidth,
+    enableToggle,
+    onToggle,
+    toggleWidth,
+    actionColSize
```
Comment (mautematico) [inline new-lamassu-admin/src/components/editableTable/Row.js:35]: this is better than what I was proposing on https://github.com/lamassu/lamassu-server/pull/390, so I'll close that.

**Pull request merged**